examples/experimental/tls-cert-cross-namespace.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         name: wildcard-example-com-cert
         namespace: gateway-api-example-ns2
 ---
-apiVersion: gateway.networking.k8s.io/v1alpha2
+apiVersion: gateway.networking.k8s.io/v1beta1
```

Comment:
Do we have any other limitations that prevent cross-namespace cert refs? I can't think of any, which makes me think this should also be labeled as "standard" now.